src/graccreq/oim/OIMTopology.py

```diff
@@ -104,7 +110,7 @@ def get_file_from_OIM():
         try:
             oim_xml = urllib2.urlopen(oim_url)
         except (urllib2.HTTPError, urllib2.URLError) as e:
-            print e
+            logging.warn(e)
```

Comment:
It kind of straddles the line for me.  Since this can happen if OIM is down briefly, or something minor like that, but might work in 15 minutes, I figure it's not something we should be overly concerned about.

I'd be happy to bump it up to error if you think that's a better idea.